spec/linter-eslint-spec.js

```diff
@@ -269,7 +273,7 @@ describe('The eslint provider for Linter', () => {
             if (messagesAfterFixing) {
               // There is still one linting error, `semi` which was disabled during fixing
               expect(messagesAfterFixing.length).toBe(1)
-              expect(messagesAfterFixing[0].html).toBe('Extra semicolon. (<a href="http://eslint.org/docs/rules/semi">semi</a>)')
+              expect(messagesAfterFixing[0].html).toBe('Extra semicolon. ([semi](http://eslint.org/docs/rules/semi))')
```

Comment:
I suspect the only reason that this didn't cause the build to fail is that this spec is rather flakey currently. This definitely should be checking against the v2 message format.